src/main/java/seedu/address/commons/util/AppUtil.java

```diff
@@ -34,9 +34,15 @@ public static void checkArgument(Boolean condition) {
      *
      * @throws IllegalArgumentException with {@code errorMessage} if {@code condition} is false.
      */
-    public static void checkArgument(Boolean condition, String errorMessage) {
+    public static void validateArgument(Boolean condition, String errorMessage) {
         if (!condition) {
             throw new IllegalArgumentException(errorMessage);
         }
     }
+    /**
+     * Checks that the {@code condition} is true, but does not throw an exception.
+     */
+    public static void checkArgument(Boolean condition, String warningMessage) {
+
+    }
```

Comment:
Perhaps there is no need to include this method when there is no use for it?

Reply:
Removed.